plugins/chunter-resources/src/components/ThreadComment.svelte

```diff
@@ -109,7 +115,7 @@
         }}
       />
     </div>
-    <div class="tool"><ActionIcon icon={Bookmark} size={'medium'} /></div>
+    <div class="tool"><ActionIcon icon={Bookmark} size={'medium'} action={pinMessage} /></div>
```

Comment:
Bookmark - save for me, pin - save for everyone. You should just create action for menu. Please look at `MarkUnread` action for example in `models/chunter/index` and `plugins/chunter-resources/index`